### PR TITLE
Unify config.yml standard

### PR DESCRIPTION
**Pull request by DFolchA**
|Related issue|
|---|
|#1045|

## Description

This PR changes the way the installer script reads the yaml files and sets a new standard for the config.yml file.

## AIO install
```
[root@centos7 unattended_scripts]# ./wazuh_install.sh -a -l -d
12/20/2021 12:08:07 INFO: Configuration file found. Creating certificates...
12/20/2021 12:08:07 INFO: Creating the Elasticsearch certificates...
12/20/2021 12:08:07 INFO: Creating Wazuh server certificates...
12/20/2021 12:08:07 INFO: Creating Kibana certificate...
12/20/2021 12:08:07 INFO: Certificates creation finished. They can be found in /home/vagrant/wazuh-packages/unattended_scripts/certs.
12/20/2021 12:08:07 INFO: Starting the installation...
12/20/2021 12:08:07 INFO: Installing all necessary utilities for the installation...
12/20/2021 12:08:08 INFO: Done
12/20/2021 12:08:08 INFO: Adding the Wazuh repository...
12/20/2021 12:08:09 INFO: Done
12/20/2021 12:08:09 INFO: Installing the Wazuh manager...
12/20/2021 12:08:51 INFO: Done
12/20/2021 12:09:02 INFO: Wazuh-manager started
12/20/2021 12:09:02 INFO: Installing Open Distro for Elasticsearch...
12/20/2021 12:10:09 INFO: Done
12/20/2021 12:10:09 INFO: Configuring Elasticsearch...
12/20/2021 12:10:19 INFO: Elasticsearch started
12/20/2021 12:10:19 INFO: Initializing Elasticsearch...
12/20/2021 12:10:27 INFO: Done
12/20/2021 12:10:27 INFO: Installing Filebeat...
12/20/2021 12:10:34 INFO: Filebeat started
12/20/2021 12:10:34 INFO: Done
12/20/2021 12:10:34 INFO: Installing Open Distro for Kibana...
12/20/2021 12:12:01 INFO: Done
12/20/2021 12:12:30 INFO: Kibana started
12/20/2021 12:12:30 INFO: Initializing Kibana (this may take a while)
12/20/2021 12:12:40 INFO: You can access the web interface https://localhost. The credentials are admin:admin
12/20/2021 12:12:40 INFO: Setting the Wazuh repository to production
12/20/2021 12:12:40 INFO: Done
```

### CODE REVIEW

File: unattended_scripts/config.yml [hunk 1]
```diff
@@ -1,21 +1,25 @@
-## Single-node configuration
+nodes:
+  elasticsearch:
+    # name: <elasticsearch-node-name>
```
Comment (verdx) [inline unattended_scripts/config.yml:3]: Why are the name and ip commented?

File: unattended_scripts/install_functions/opendistro/elasticsearch.sh [hunk 2]
```diff
@@ -97,63 +97,45 @@ configureElasticsearch() {
 
     checkNodes
```
Comment (verdx) [inline unattended_scripts/install_functions/opendistro/elasticsearch.sh:98]: If you do not use the single anymore, this line should be erased

File: unattended_scripts/install_functions/opendistro/wazuh-cert-tool.sh [hunk 3]
```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-
+# set -x
```
Comment (verdx) [inline unattended_scripts/install_functions/opendistro/wazuh-cert-tool.sh:2]: This shouldn't stay here.

**Review by verdx — CHANGES_REQUESTED** [review]
GJ, only some minor changes

**Review by verdx — APPROVED** [review]
LGTM

**Review by alberpilot — APPROVED** [review]
LGTM

**Pull request merged**